docs/cr.yaml

```diff
@@ -0,0 +1,2 @@
+pages-branch: cspl-1985
```

Comment:
Hope it is removed before the merge?

Reply:
Yes, branch needs to main since the docs are currently hosted off the main branch.